src/cpu.js

```diff
@@ -974,7 +981,25 @@ CPU.prototype.init = function(settings, device_bus)
     }
 };
 
-CPU.prototype.load_multiboot = function(buffer)
+CPU.prototype.load_multiboot = function (buffer)
+{
+    if (this.bios.main) {
+        dbg_assert(false, "load_multiboot not supported with BIOS");
+    }
+
+    const { option_rom } = this.load_multiboot_option_rom(buffer, false, "");
+    if(option_rom)
+    {
+        dbg_log("loaded multiboot", LOG_CPU);
+        this.write_blob(new Uint8Array(option_rom.data), 0);
```

Comment:
I believe the `new Uint8Array` makes an unnecessary copy of the option rom and can be removed.